src/spec-node/singleContainer.ts

```diff
@@ -193,6 +193,9 @@ async function buildAndExtendImage(buildParams: DockerResolverParameters, config
 				args.push('--load'); // (short for --output=docker, i.e. load into normal 'docker images' collection)
 			}
 		}
+		if (buildParams.buildxCacheTo) {
+			args.push('--cache-to', buildParams.buildxCacheTo);
```

Comment:
Do you think it would make sense to include this passing this flag in the `src/spec-node/docker-compose.ts` case too?

Somewhere around here perhaps? https://github.com/devcontainers/cli/blob/main/src/spec-node/dockerCompose.ts#L235

Reply:
Thank you for comment!

- Devcontainer uses `docker compose build` for building image from `compose.yaml`. However, `docker compose build` does not have `--cache-to` option.
- Instead, it may make sense to include `cache-to` section in overriden docker compose files which devcontainer command renders.
- In my recoginition, devcontainer does not support `--push` option when building from docker compose config file.
- `--cache-to` option is usually used with `--push` option in CI. So, it is a little wired that `--cache-to` is supported, but `--push` is not.

We don't need to add `--cache-to` for Docker Compose cases  as my conclusion.
Objections welcome! Please let me know if there are any misunderstanding.

Reply:
Thanks for the detailed response, makes sense to me now.